### PR TITLE
feat(icon): astro

### PR DESCRIPTION
**Pull request by Eveeifyeve**
Adds an icon for astro.

### CODE REVIEW

**Comment by Eveeifyeve** [comment]
I haven't tested it yet but it should just work.

File: assets/language/astro.png
Comment (vyfor) [inline assets/language/astro.png:1]: Hey! Could you please enlarge the logo and change the background to a solid color?
I usually size the logo to be about 45-50% of the canvas. It appears a bit tiny as is:
image

**Review by vyfor — CHANGES_REQUESTED** [review]

File: assets/language/astro.png
Comment (Eveeifyeve) [reply assets/language/astro.png:1]: > Hey! Could you please enlarge the logo and change the background to a solid color? I usually size the logo to be about 45-50% of the canvas. It appears a bit tiny as is: image

Could you give me a color that would look good?

Comment (Eveeifyeve) [reply assets/language/astro.png:1]: I have fixed the size at [9211a91](https://github.com/vyfor/cord.nvim/pull/111/commits/9211a91bd2a58afd8d8384f4d4506a0a2cdde72d).

Comment (Eveeifyeve) [reply assets/language/astro.png:1]: > And please squash your commits when you're done 🙂

I will look into squashing all of these commits once done the changes in this comment: https://github.com/vyfor/cord.nvim/pull/111#discussion_r1774803824

Comment (Eveeifyeve) [reply assets/language/astro.png:1]: I have now squashed all commits.

**Comment by Eveeifyeve** [comment]
I missed a bit on the left side of the icon, I am going to push and squash a commit to fix this.

**Review by vyfor — APPROVED** [review]

**Comment by vyfor** [comment]
All good, thanks!

**Pull request merged**